Library/Formula/openjpeg.rb

```diff
@@ -22,7 +22,13 @@ class Openjpeg < Formula
   depends_on "libpng"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    args = std_cmake_args
+    system "cmake", ".", *args
+    system "make", "install"
```

Comment:
Mmm I'd really like to grab the static libraries from the bottles. I can try to run `make` twice and `make install` only once?